glossary.dd

```diff
@@ -258,6 +258,11 @@ void test()
         syntax) refers to the ability to call a function as if it were a method of
         its first argument. For example `funct(arg)` may be written as
         `arg.funct()`.)
+
+        $(DT $(LNAME2 inclusive_destructor, inclusive-destructor, Inclusive
+        Destructor))
+        $(DD An inclusive destructor is a destructor that includes destruction
+        of the object itself and its fields, if reaching its lifetime.)
```

Comment:
```suggestion
        of the object itself and its fields.)
```
Destructors can be called early (for example, by `destroy`), and this definition applies regardless of when it is called.